code/modules/power/port_gen_vr.dm

```diff
@@ -29,6 +29,7 @@
 	req_components = list(
 		/obj/item/stack/cable_coil = 5,
 		/obj/item/weapon/stock_parts/capacitor = 1,
+		/obj/item/weapon/stock_parts/micro_laser = 1,
```

Comment:
Why a micro laser and not just... two capacitor slots? Most machines usually have different parts have different effects.

Reply:
Same as source